_service

```diff
@@ -11,7 +11,6 @@
     <param name="versionrewrite-pattern">v(\d+\.\d+\.\d+)</param>
     <param name="versionrewrite-replacement">\1</param>
     <param name="changesgenerate">enable</param>
-    <param name="package-meta">yes</param>
```

Comment:
The idea is to have a copy of the history, so I'd prefer to rename the .git during build. So instead add to the end of %prep something like:
mv .git .git-deactivated

Reply:
done. this worked too.